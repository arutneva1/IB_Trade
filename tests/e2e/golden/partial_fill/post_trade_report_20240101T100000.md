| symbol | side | filled_shares | avg_price | notional | avg_slippage | residual_drift_bps |
| --- | --- | --- | --- | --- | --- | --- |
| AAA | SELL | -5.0000 | 99.00 | -495.00 | 0.00 | -5000.00 |
