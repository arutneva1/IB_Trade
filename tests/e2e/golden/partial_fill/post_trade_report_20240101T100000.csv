symbol,side,filled_shares,avg_price,notional,avg_slippage,residual_drift_bps
AAA,SELL,-5.0,99.0,-495.0,0.0,-5000.0
